GauchoGrub/app/src/main/java/edu/ucsb/cs/cs190i/gauchogrub/gauchogrub/FavoriteRecyclerAdapter.java

```diff
@@ -0,0 +1,140 @@
+package edu.ucsb.cs.cs190i.gauchogrub.gauchogrub;
```

Comment:
It definitely is, I looked into it at some point and had some imports flip out, so I didn't do it then. I'll take a look again now.